sklearn/linear_model/_glm/link.py

```diff
@@ -0,0 +1,114 @@
+"""
+Link functions used in GLM
+"""
+
+# Author: Christian Lorentzen <lorentzen.ch@googlemail.com>
+# License: BSD 3 clause
+
+from abc import ABCMeta, abstractmethod
+
+import numpy as np
+from scipy.special import expit, logit
+
+
+class Link(metaclass=ABCMeta):
```

Comment:
I would have called this class BaseLink to explicit it's a base class that is not meant to be instantiated directly